whois-query/src/main/java/net/ripe/db/whois/query/pipeline/WhoisServerChannelInitializer.java

```diff
@@ -59,11 +66,20 @@ public WhoisServerChannelInitializer(final MaintenanceHandler maintenanceHandler
         this.connectionPerIpLimitHandler = connectionPerIpLimitHandler;
         this.queryHandler = queryHandler;
         this.applicationVersion = applicationVersion;
+
```

Comment:
minor: Any reason for splitting the logic between the constructor and `initChannel` method? It looks to be all the logic can go into `initChannel`. check if `proxyProtocolEnabled`, log if so, and add to the channel pipeline. I think that reads simpler (since it is in one place) and no need to add a member variable (proxyProtocolChannelHandler)

Reply:
I'd still need to pass in the feature toggle via the constructor and keep it as a instance variable (can't inject it into initChannel() method).

Reply:
Then maybe only add have the feature toggle assignment in the constructor...and it's usage and other logic...in initChannel.

But I'll say this is more of a _sylistic_ remark, so it is fine to have it as is...since it works as it should

Reply:
Done.